docs/operations/release-process.md

```diff
@@ -0,0 +1,78 @@
+# Release Process
```

Comment:
Cleanup this documentation, especially I dont think we agreed / discussed the roles section.